airflow/www/views.py

```diff
@@ -1354,6 +1356,82 @@ def xcom(self, session=None):
             title=title,
         )
 
+    @expose('/task_note', methods=['POST', 'GET'])
```

Comment:
If you are creating a TaskNoteModelView then  we shouldn't have this method here -- but use the model view. Having both is wrong.

Reply:
I am not sure I understand this right. 
I took inspiration from XCOM which has an `XComModelView` and a dedicated `xcom` component in the views (xcom is an old component and I should not have followed that?). 

Should I expose these two endpoints with two different functions in the TaskNoteModelView?

Reply:
What I hand in mind is to have two "views"
- One in the menu whose endpoint is "task_notes" (screenshot here https://ibb.co/VCVcrSp) which is modeled by `TaskNoteModelView` where users can have a summary of all notes and can edit/delete/add them. 
- At the same time I am proposing to extend the "Task instance" view with a task note view (endpoint /task_note) screen https://ibb.co/CWdMBkb where a user can post and get all the notes related to specific task execution. This would require a different endpoint and a different view from what I can understand. Is that not the case? 😄